packages/core/utils/src/http-server.js

```diff
@@ -7,7 +7,8 @@ import type {FilePath, HTTPSOptions} from '@parcel/types';
 import type {FileSystem} from '@parcel/fs';
 
 import http from 'http';
-import https from 'https';
+// $FlowFixMe Flow does not know of http2's existance
```

Comment:
I assume we don't need to install anything for this and that it'll just happen to come with Flow some day?

Reply:
Correct